app/views/messages/index.html.erb

```diff
@@ -1,8 +1,10 @@
 <%= content_for :messages_content do %>
 
+  <%= turbo_stream_from @conversation %>
```

Comment:
I moved the stream_from listener out of the `two_columns` and into this in order to fix a bug. In the old place, when you load the page and click to a new conversation on the left side, it would not reconnect the stream_from listener. When I moved it inside here, now it continues to stream in new responses even when you change conversations.